recipes/boost-ext-ut/all/conanfile.py

```diff
@@ -69,4 +69,4 @@ def package_info(self):
         self.cpp_info.names["cmake_find_package_multi"] = "boost"
         self.cpp_info.filenames["cmake_find_package"] = "ut"
         self.cpp_info.filenames["cmake_find_package_multi"] = "ut"
-        self.cpp_info.components["ut"].includedirs = []
+        self.cpp_info.components["ut"].includedirs = ['include']
```

Comment:
I am fairly certain this is the default

Reply:
I think so too. Can't you just remove the line?
Lise so:
```suggestion
```

Reply:
I thought the default might be different for components, but you are correct that it is the default!
I can't remove the whole line, because I still need to specify the `ut` component, but I have removed the `includedirs` directive to avoid duplicating the default behavior.